.circleci/config.yml

```diff
@@ -171,6 +184,97 @@ jobs:
           path: /coverage
           desination: coverage
 
+  regression_test:
+    docker:
+      - image: circleci/python:3.7.4
+    working_directory: ~/repo
+    steps:
+      - run: cd ~; git clone https://github.com/lightstep/lightstep-benchmarks.git repo
+      - run: ./scripts/ci_setup.sh
+      - attach_workspace:
+          at: ~/plugin
+      - run: sudo pip install ~/plugin/*.whl
+      - run: pytest -s --client_name python-cpp regression_tests.py
+      - store_artifacts:
+          path: logs
+          destination: logs
+
+  make_cpu_graphs:
+    docker:
+      - image: circleci/python:3.7.4
+    working_directory: ~/repo
+    steps:
+      - run: cd ~; git clone https://github.com/lightstep/lightstep-benchmarks.git repo
```

Comment:
Could you clone the lightstep-benchmark repo to a specific commit so that the build won't break if the master branch ever changes in an incompatible way?